unit_tests/test_version_utils.py

```diff
@@ -1,16 +1,19 @@
 from __future__ import absolute_import
-import unittest
 
-from sdcm.utils.version_utils import get_branch_version_from_list, get_branch_version_from_repo, get_branch_version, is_enterprise
+import unittest
 
-# pylint: disable=invalid-name
+from sdcm.utils.version_utils import get_branch_version_from_list, get_branch_version_from_repo, get_branch_version, \
+    is_enterprise, VERSION_NOT_FOUND_ERROR
 
-DEB_URL = 'https://s3.amazonaws.com/downloads.scylladb.com/enterprise/deb/unstable/stretch/'\
-          '9f724fedb93b4734fcfaec1156806921ff46e956-2bdfa9f7ef592edaf15e028faf3b7f695f39ebc1'\
-          '-525a0255f73d454f8f97f32b8bdd71c8dec35d3d/68/scylladb-2019.1/scylla.list'
-RPM_URL = 'https://s3.amazonaws.com/downloads.scylladb.com/enterprise/rpm/unstable/centos/'\
-          '9f724fedb93b4734fcfaec1156806921ff46e956-2bdfa9f7ef592edaf15e028faf3b7f695f39ebc1'\
-          '-525a0255f73d454f8f97f32b8bdd71c8dec35d3d-a6b2b2355c666b1893f702a587287da978aeec22/71/scylla.repo'
+BASE_S3_DOWNLOAD_URL = 'https://s3.amazonaws.com/downloads.scylladb.com'
+DEB_URL = \
```

Comment:
For some reason, the `pre-commit` or `Pylint` tries to arrange the length of the row.
Therefore, after arranging we get that the first row is not on the same line if the other rows.